Quick-Start [DA]/MVP/MVP+Makefile.md

```diff
@@ -1 +1,274 @@
+# Minimum Viable Product:
+A basic tutorial on creating a PostgreSQL extension using the bare minimum files and a Makefile:
+
+- A **control file**, which tells PostgreSQL some basic information about the extension, such as its name, version, and schema.
+- A **SQL script file**, which contains the SQL commands to create the extension's objects.
+- A **Makefile** is an optional but highly recommended file that helps automate the build process of the extension. It defines the compilation and installation steps, making it easier to build and install the extension on different systems.
+
+# [CODE](https://github.com/IshaanAdarsh/Postgres-extension-tutorial/tree/main/Code/my_extension)
+
+## A Postgres extension using only a control file, an SQL file and a Makefile:
+
+### Step 1: Create the Extension Directory
+- Create a directory for your extension. Let's call it "my_extension".
+
+```bash
+$ mkdir my_extension
+$ cd my_extension
+```
+
+### Step 2: Create the Extension Control File
+- Inside the "my_extension" directory, create a file named "my_extension.control". 
+  - This file defines the metadata and dependencies of your extension.
+
+```bash
+$ touch my_extension.control
+```
+
+- Open the "my_extension.control" file and add the following content:
+
+```control
+# my_extension.control
+comment = 'Minimal Viable Product'
+default_version = '1.0'
+relocatable = true
+
+# Installation script for version 1.0
+# 1.0
+# my_extension--1.0.sql
+```
+
+### Step 3: Create the Extension SQL File
+- Create a file named "my_extension--1.0.sql".
+  - This file contains the SQL statements required to install or uninstall your extension.
+
+```bash
+$ touch my_extension--1.0.sql
+```
+
+- Open the "my_extension--1.0.sql" file and add the following content:
+
+```sql
+-- extension--1.0.sql
+-- my_extension--1.0.sql
+-- Create necessary objects for version 1.0
+CREATE TABLE my_table (
+  id SERIAL PRIMARY KEY,
+  name VARCHAR(100) NOT NULL
+);
+```
+
+### Step 4: Create the Makefile
+- Create a Makefile in the "extension" directory. 
+  - The Makefile is used to define the build process. It specifies the module name, object files, extension name, and SQL files. The PG_CONFIG variable retrieves the path to the pg_config utility, which is used to get the necessary configuration for building the extension.
+
+```bash
+$ touch Makefile
+```
+- Open the Makefile and add the following content:
+<!--Problem: https://dba.stackexchange.com/questions/328624/makefile-issue-in-creating-postgres-extension
```

Comment:
Since you've solved this problem, do you need to keep this comment?

Reply:
These comments are just for my reference, just to make sure that if i run into these issues again i know what the problem and the solution is